packages/user-ui/src/pages/projects/_id/components/comments.rs

```diff
@@ -1,8 +1,243 @@
+                    }
+                    div { class: "w-full min-h-[40px] flex justify-between",
+                        div { class: "w-full min-h-[20px] flex flex-row justify-start items-end gap-[20px]",
+                            //quotation button
+                            button {
+                                class: "w-[24px] h-[24px] flex justify-center items-center cursor-pointer",
+                                onclick: move |_| {
+                                    println!("quotation button clicked");
+                                },
+                                SquareMark { color: "#555462" }
+                            }
+                            //code button
+                            button {
+                                class: "w-[24px] h-[24px] flex justify-center items-center cursor-pointer",
+                                onclick: move |_| {
+                                    println!("code button clicked");
+                                },
+                                Code { color: "#555462" }
+                            }
+                            //attach image button
+                            button {
+                                class: "w-[24px] h-[24px] flex justify-center items-center cursor-pointer",
+                                onclick: move |_| {
+                                    println!("attach image button clicked");
+                                },
+                                AddPhoto { color: "#555462" }
+                            }
+                            //attach file button
+                            button {
+                                class: "w-[24px] h-[24px] flex justify-center items-center cursor-pointer",
+                                onclick: move |_| {
+                                    println!("attach file button clicked");
+                                },
+                                Link2 { color: "#555462" }
+                            }
+                            //font style bold button
+                            button {
+                                class: "w-[24px] h-[24px] flex justify-center items-center cursor-pointer",
+                                onclick: move |_| {
+                                    println!("font style bold button clicked");
+                                },
+                                BoldIcon { color: "#555462" }
+                            }
+                        }
+                        // comment summit button
+                        button {
+                            class: "w-[40px] h-[40px] flex justify-center items-center bg-[#EBEFF5] rounded-full cursor-pointer",
+                            onclick: move |_| submit_reply(),
+                            SquareChat { color: "#8095EA" }
+                        }
+                    }
+                }
+            }
+
+            if (*show_comments)() {
```

Comment:
```suggestion
            if show_comments() {
```
dioxus 문서 한번은 꼭 읽으시길 바랍니다.